### PR TITLE
Unit test for the delete button.

### PR DESCRIPTION
**Pull request by AsmahanM**
I've added a test to see if an item has or hasn't been removed from the list.

The test is not showing any errors in the console. However, it is also not indicating whether the test has passed or not so I am not sure if this is a false positive.

### CODE REVIEW

File: index-test.js [hunk 1]
```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
...
+    const result = movieDiv.includes(movietitle[0])
+    const expected = false
+    equal (result, expected);
+    deleteBtn[0].click();
+  })
+  }, 5000);
+  
+
+setTimeout(() => { //Allows time for the elements to appear on the page as they're not present onload.
+test("deleteItem() function should remove an item from the list", () => {
```
Comment (PJSalter) [inline index-test.js:175]: Great work on the delete button testing, very clear to read 🍿 📺 🥳

File: index-test.js [hunk 2]
```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
...
+  const movieDiv = document.querySelectorAll('.movie-name') //Outer container
+  const movieTitle = document.querySelectorAll('.movie-title') //Input value
+  const deleteBtn = document.querySelectorAll('button') //delete button
+  if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
+  let result = deleteBtn[0].click(); //When the first delete button is clicked...
+  const expected = deleteItem(movieDiv.contains(movieTitle[0]) == false); //...the first input will no longer be contained in the div.
```
Comment (PJSalter) [inline index-test.js:181]: I like that you use comments throughout your code, clearly described 😸 💯

File: index-test.js [hunk 3]
```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
   result.textContent = "";
 });
 
-*/
\ No newline at end of file
+*/
+
+setTimeout(() => {
+
+  test("deleteItem() function should remove an item from the list", () => {
```
Comment (PJSalter) [inline index-test.js:160]: Yaaaaaaay 🎉 🥳  we have more tests for our binge list project, great work here ✨ 

very neat 😄

**Review by PJSalter — APPROVED** [review]

File: index-test.js [hunk 4]
```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
...
+    const deleteBtn = document.querySelectorAll('button');
+    const movieDiv = document.querySelectorAll('.movie-name');
+    const movietitle = document.querySelectorAll('.movie-title');
+    if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
+    deleteBtn[0].click(); //When the first delete button is clicked...
```
Comment (mariaalouisaa) [inline index-test.js:165]: Nice comments! Makes it clear whats going on 🙌

File: index-test.js [hunk 5]
```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
...
\ No newline at end of file
+*/
+
+setTimeout(() => {
+
+  test("deleteItem() function should remove an item from the list", () => {
+    const deleteBtn = document.querySelectorAll('button');
+    const movieDiv = document.querySelectorAll('.movie-name');
+    const movietitle = document.querySelectorAll('.movie-title');
+    if(this.item === undefined) {return} //This stops the .click() from returning an error message.
```
Comment (mariaalouisaa) [inline index-test.js:164]: I still don't quite get this, but will read up on it now...

File: index-test.js (hunk 2 above)
Comment (mariaalouisaa) [reply index-test.js:181]: Same! Very tidy

**Review by mariaalouisaa — APPROVED** [review]

**Pull request merged**